src/averageAges.js

```diff
@@ -14,12 +14,21 @@
  *
  * @return {number}
  */
+
+const isObject = (value) => typeof value === 'object' && value !== null;
+
+const getAverageAge = people => isObject(people[0])
```

Comment:
The check is here - that's good

However, you are a bit inconsistent
if you are trying to see that the `people` array is not only empty but also has some meaningful data - it makes sense also check the `born` and `died` properties. But that looks like overkill for the current task

I suggest you simply check if `people.length` is greater than 0.